README.md

```diff
@@ -1,3 +1,79 @@
-Open Source Church WebApp Template. 
+# Open Source Church WebApp Template
```

Comment:
@dfout can you also please include the discord link?  This is my thinking -  It would be best for our project to include all our contributors in README file. This can be achieved using GitHub action https://github.com/marketplace/actions/auto-add-contributors. Please let me know if you can look into this. Thanks

Reply:
Sure thing, I can have that done by EOD PST.